src/ClientModel/CaptureData.cpp

```diff
@@ -175,15 +175,21 @@ const std::string& CaptureData::GetFunctionNameByAddress(uint64_t absolute_addre
 // callstack.
 std::optional<uint64_t> CaptureData::FindFunctionAbsoluteAddressByAddress(
```

Comment:
Kind of unrelated to this change, but I find the name (including the parameter name) pretty confusing. Maybe we can use the opportunity to improve here.

Reply:
Do you have a name in mind?

Reply:
Yeah, it's confusing, looking at the name it's almost like it just returns the argument. Maybe `FindFunctionAbsoluteAddressByInstructionAbsoluteAddress` (if I even got it correctly)?

Reply:
done.